<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWidget</class>
 <widget class="QWidget" name="AdminWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>728</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>管理面板</string>
  </property>
  <widget class="QPushButton" name="m_User">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>用户管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_Course">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>课程管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_Grade">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>成绩管理</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
